yang-lsp/gradle/versions.gradle

```diff
@@ -1,8 +1,8 @@
 version = '0.1.0-SNAPSHOT'
 
 ext.versions = [
-    xtext: '2.15.0-SNAPSHOT',
```

Comment:
Do we need the SNAPSHOT build?

Reply:
We need `2.16.0` for sure; it contains the changes for the hierarchical document symbols. I do not know if we can have `2.16.0` without the snapshot.